java/src/test/java/ai/rapids/cudf/CudfTestBase.java

```diff
@@ -108,4 +109,35 @@ static void afterAll() {
     }
     LISTENER.checkOutstanding();
   }
+
+  private static boolean doublesAreEqualWithEpsilon(double expected, double actual, double epsilon) {
+    if (Double.doubleToLongBits(expected) != Double.doubleToLongBits(actual)) {
```

Comment:
I don't see the value in this check.  It is extra code that does not appear to add any value.

Reply:
We need that check for equality test of NaNs, +ve infin and -ve infin. I have documented the code to be explicit